THELIST.md

```diff
@@ -0,0 +1,7 @@
+# The List of Brokers
```

Comment:
What about putting it on the README so that it's the first thing you see?

Reply:
I want to make sure the first thing people see is some context, the code of conduct, etc... I've moved the "want to jump right to the list?" part up. Thoughts?

Reply:
How about at the bottom of the readme? 😇

Reply:
fair nuff! will do